packages/blockchain-wallet-v4-frontend/src/components/CopyClipboardButton/index.tsx

```diff
@@ -0,0 +1,59 @@
+import * as C from 'services/AlertService'
+import { actions } from 'data'
+import { bindActionCreators } from 'redux'
+import { connect, ConnectedProps } from 'react-redux'
+import CopyClipboard from './template'
+import React from 'react'
+
+export interface OwnProps {
+  address: string
+  alertActions: any
+}
+
+export interface State {
+  active: boolean
+}
```

Comment:
I don't think it really matters if we use `interface` or `type` for props, but since we use `type` everywhere else I would urge you to use that for the sake of consistency. We also tend place types below the component because they depend on `const connector` and this way they are all together. 👇